examples/terraform-azure-aks-example/main.tf

```diff
@@ -0,0 +1,55 @@
+# ---------------------------------------------------------------------------------------------------------------------
```

Comment:
NIT: I know many of our examples in this repo don't do this, but for this module it might help to have a preamble with a high level description of what might be deployed. This is because there are more than 2 resources here, so it helps to give some context on the end result. See https://github.com/gruntwork-io/terraform-aws-influx/blob/master/examples/tick-multi-cluster/main.tf#L1 for example (this is one of our public modules that we maintain at Gruntwork).

Reply:
NIT: We use two space tabbing for our tf files. It would be good to be consistent.

Reply:
Done.  Please review it.